rpgJavaInterpreter-core/src/test/resources/ARRAYDS.rpgle

```diff
@@ -0,0 +1,15 @@
+     V* ==============================================================
+     V* 08/04/2025 APU002 Creation
+     V* ==============================================================
+    O * PROGRAM GOAL
+    O * Perform a CALL passing an array value to a field received as DS
+     V* ==============================================================
+    O * JARIKO ANOMALY
+    O * Before the fix, jariko crashed
+     V* ==============================================================
+     D £C6MIM          DS
+     D  £6I                          20P 6 DIM(99) INZ
+     C                   CALL      'ARRAYDS_1'
```

Comment:
Declare an array of 2 elements, initialize only the first one, and check both elements on return.

Try on AS400 because you are passing a field type (array) different from the type declared in ARRAYDS\_1 entry.

Possibly it's correct, but I want to be sure.